homeassistant/components/fujitsu_fglair/strings.json

```diff
@@ -4,12 +4,12 @@
       "user": {
         "title": "Enter your FGLair credentials",
         "data": {
-          "is_europe": "Use european servers",
+          "region": "Region",
+          "other": "Other",
+          "europe": "Europe",
+          "china": "China",
```

Comment:
That's not how a selector gets translations, it works via a separate `selector` key